### DISCUSSION TITLE
RTL and LTR setup in a Vite Vue project

### DISCUSSION THREAD

**Discussion by RejownAhmed** [twbs/bootstrap#39512] [Q&A]
Hi, I am developing a vue website with bootstrap. I need rtl and ltr both functionality. Unfortunately, I don't find any good solutions anywhere, even the bootstrap doc is not very clear about it. Can you someone please help in this case?

**Comment by 3baaady07** [comment]
Consider using [`postcss`](https://postcss.org/) and integrate the [`postcss-rtlcss`](https://github.com/elchininet/postcss-rtlcss) plugin by adding a `postcss.config.js` file and add configurations to it.

Here's a basic setup,

```javascript
// postcss.config.js
import postcssRTLCSS from 'postcss-rtlcss';

export default {
   plugins: {
      ['postcss-rtlcss']: postcssRTLCSS(),
   },
}

```

**Reply by RejownAhmed** [reply]
Thanks, let me check

**Reply by 3baaady07** [reply]
Here's an updated config code. Notice the omitted `postcssRTLCSS()` as I came to know that it is not needed. Please note that you need `postcss-rtlcss` for this to work. Run the command `npm i -D postcss-rtlcss`.
```js
// postcss.config.js

export default {
   plugins: {
      ['postcss-rtlcss']: {},
   },
}
```
